### DISCUSSION TITLE
Combobox cell causing re-render which is closing the popover and user is unable to type in input field

### DISCUSSION THREAD

**Discussion by sudarshan-mane** [TanStack/table#5495] [Q&A]
As per doc-

We have Memoize column and data both,
All the components inside column are created separate component outside table.
Issue - -We have table cell which is combobox. Combobox has search input and options. -When we type anything in search input, using next.js server action, we get options from backend. So, after typing every letter, options are changing, which cause table re-renders.

Issue Video - https://www.loom.com/share/9f173bd1249041d78c11773e8f8d265e?sid=3210ddaa-8e0d-43ee-8d73-c5d3293e7bad

Code Sandbox: https://codesandbox.io/p/github/rkandwal300/tanstack-table-issue/master

We are stuck with the issue since a week, we appreciate any help. Thanks in Advance.
We haven't found single example with Combobox/Autocomplete, which fetches options from backend.

**Comment by laGameTV** [comment]
Did you find a solution? I'm currently facing the same problem.
